src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -120,6 +120,27 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                             IsAzureDB = isAzureDB
                         };
 
+                        // Collect the Database properties information
+                        if (!requestParams.IsNewObject)
+                        {
+                            var smoDatabaseProperties = dataContainer.SqlDialogSubject as Database;
+                            databaseViewInfo.ObjectInfo = new DatabaseInfo()
+                            {
+                                Name = smoDatabaseProperties.Name,
+                                CollationName = smoDatabaseProperties.Collation,
+                                DateCreated = smoDatabaseProperties.CreateDate.ToString(),
+                                LastDatabaseBackup = smoDatabaseProperties.LastBackupDate == DateTime.MinValue ? "None" : smoDatabaseProperties.LastBackupDate.ToString(),
```

Comment:
Do any of the user facing strings like "None" need to be localized?

Reply:
Need to check with other language installers, but do we have any localized files maintained in STS, or do declaring 'const string' variables automatically take care of localization?

Reply:
Yes, new strings get added in sr.strings here: https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

When you do a full build the localization files get generated (and have to be checked in). You can then use localized strings with the SR class, like SR.BasicAzureEdition for the basic edition name for an Azure database, for example.

Reply:
See the note here : https://github.com/microsoft/sqltoolsservice/wiki#building

It would be great to add more explicit documentation around the process of adding localized strings - so if this is something you two could work on that'd be fantastic.

Reply:
I was just able to add a localized string and will be adding the process to the doc! @corivera, please update it with your insights too!